### ISSUE TITLE
addition of a point to point entry type/category to allow people to discuss route optimization for busses

### ISSUE THREAD

**Issue by zpallin** [codeforsanjose/MobilityMap#56]
Inspired by a comment from DevPost viewable [here](http://devpost.com/software/mobility-map/updates/37076?ref_content=update-comment&ref_feature=update-comment&ref_medium=email&utm_campaign=update-comment&utm_content=new_update_comment&utm_medium=email&utm_source=transactional).

> [Duende Tao](http://devpost.com/duende)
> Great project and I really like the simplicity and accessibility for the end user. If possible, I would like to see an addition of a point to point entry. Mobility Map could perhaps already include that feature and I did not see it in the video, however an ability for users to make constructive criticism on a whole route, or point to point would assist clarity in many cases and be useful indeed. Keep up the good work!

**Comment by jamestyack** [comment]
Any thoughts on how this could work Zach? Perhaps needs some discussion. I'm not too familiar with all the features of the app as it stands but as a starting point would it make sense to have option to select a bus route (populated by GTFS routes data?) and then comment on that route? I'm not sure if you're thinking that map should show routes somehow (or maybe this could be optional based on filter).

**Comment by zpallin** [comment]
@jamestyack 
I think I would prefer a more limited scope on this to perhaps reduce time to implement. 

What I had in mind was part of a general plan to have custom fields that associate with particular categories. The example is this case is that there would be a specific bus category that would allow you to place transfer point suggestions and perhaps detail times that it would be useful to have buses there. Maybe I'll allow other people who tag on to throw in additional times that they think would be good. 

Let me know what you think.

**Comment by jamestyack** [comment]
Makes sense Zach. Curious if you've looked at MBTA Ninja .... there may be some UX patterns there worth looking. Slightly different use case but I see similarities on the reporting side.

**Comment by zpallin** [comment]
@jamestyack I looked at it before I started this project, so maybe I should go back and get some more ideas. Thanks for the heads up!
